### PR TITLE
documentation update - docker use port 3000

### PR DESCRIPTION
**Pull request by navalev**

### CODE REVIEW

**Review by SharonHart — DISMISSED** [review]

File: docs/anonymizer/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Persidio anonymizer comes with predefined anonymizers but can easily be extended
     docker pull mcr.microsoft.com/presidio-anonymizer
     
     # Run the container with the default port
-    docker run -d -p 5001:5001 mcr.microsoft.com/presidio-anonymizer:latest
+    docker run -d -p 5001:3000 mcr.microsoft.com/presidio-anonymizer:latest
```
Comment (omri374) [inline docs/anonymizer/index.md:33]: Do we want both to have the same default port?

Comment (navalev) [reply docs/anonymizer/index.md:33]: currently the default port for each service is 3000 (set in app.py). we set it to a different one in the dockercompose.yml, but when running it like this - the port is 3000, redirected to 5001 (or whatever port we want). otherwize this does not work

Comment (omri374) [reply docs/anonymizer/index.md:33]: Sorry if I wasn't clear. Now both the analyzer and the anonymizer redirect from 3000 to 5001. Do we want the anonymizer to redirect to 5002?

Comment (navalev) [reply docs/anonymizer/index.md:33]: got it, fixed

**Review by omri374 — APPROVED** [review]

**Pull request merged**